package.json

```diff
@@ -8,6 +8,8 @@
   "dependencies": {
     "@material-ui/core": "^4.2.0",
     "@material-ui/icons": "^4.2.1",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
Can I suggest to add `"import/no-extraneous-dependencies": ["error", { "devDependencies": true }]` as Eslint rule because as we see here It transferred `enzyme` libraries from devDependencies to dependencies.